lint/peglint.cc

```diff
@@ -98,29 +110,9 @@ int main(int argc, const char** argv)
         cerr << source_path << ":" << ln << ":" << col << ": " << msg << endl;
     };
 
-    if (opt_trace) {
```

Comment:
I want to keep this code here, because this trace format is not for general purpose, but only for peglint tool.